src/libaktualizr/uptane/uptane_network_test.cc

```diff
@@ -113,6 +115,30 @@ TEST(UptaneNetwork, no_errors_sqlite) {
   EXPECT_TRUE(doTestInit(StorageType::kSqlite, "noerrors", "noerrors"));
 }
 
+TEST(UptaneNetwork, DownloadFailure) {
+  TemporaryDirectory temp_dir;
+  conf.storage.path = temp_dir.Path();
+  conf.provision.expiry_days = "noerrors";
+  conf.provision.primary_ecu_serial = "noerrors";
+  conf.provision.primary_ecu_hardware_id = "download_failure";
```

Comment:
Is there a significance to "download_failure" here? I don't see it used elsewhere.